pkg/koordlet/statesinformer/states_noderesourcetopology.go

```diff
@@ -307,14 +361,16 @@ func (s *nodeTopoInformer) reportNodeTopology() {
 		return
 	}
 
-	ctx := context.TODO()
 	node := s.nodeInformer.GetNode()
 	err = util.RetryOnConflictOrTooManyRequests(func() error {
 		var nodeResourceTopology *v1alpha1.NodeResourceTopology
 		if features.DefaultKoordletFeatureGate.Enabled(features.NodeTopologyReport) {
-			nodeResourceTopology, err = s.topologyClient.TopologyV1alpha1().NodeResourceTopologies().Get(ctx, node.Name, metav1.GetOptions{ResourceVersion: "0"})
-			if err != nil {
-				klog.Errorf("failed to get node resource topology %s, err: %v", node.Name, err)
+			nodeResourceTopology, err = s.nodeResourceTopologyLister.Get(node.Name)
+			if errors.IsNotFound(err) {
+				klog.Warningf("nodeTopo %s not found, skip", node.Name)
```

Comment:
The NRT is asserted to be created if we reach here, so we should not ignore the ErrNotFound. Also, it is not recommended to log "skip" message in WARNING level.